lib/wallet/widgets/dialogs/account_dialog.dart

```diff
@@ -170,8 +170,9 @@ class AccountDialog extends StatelessWidget {
                         ],
                       ),
                     ),
-                    Row(
-                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                    Wrap(
+                      //Switched Row to Wrap to prevent overflowing
+                      direction: Axis.vertical,
```

Comment:
I don't think its a good idea to have a wrap around this. Our mockup has those two buttons adjacent to each other. Also,  this design was already in place for the mockup that we are using. Also please remove the comments.